beacon-chain/core/feed/state/events.go

```diff
@@ -45,3 +48,11 @@ type InitializedData struct {
 	// GenesisValidatorsRoot represents ssz.HashTreeRoot(state.validators).
 	GenesisValidatorsRoot []byte
 }
+
+// ReorgData is the data alongside a reorg event.
+type ReorgData struct {
+	// NewSlot is the slot of new state after the reorg.
+	NewSlot uint64
```

Comment:
Is this correct ? shouldn't it be new block root instead

Reply:
This is fine. `ReorgData` is just to notify downstream service and based on downstream service's need. Currently `OldRoot` and `NewRoot` are not needed by the downstream services. They can be added later if its necessary